packages/financial-templates-lib/src/price-feed/CreatePriceFeed.ts

```diff
@@ -50,8 +50,8 @@ export async function createPriceFeed(
   config: any
 ): Promise<PriceFeedInterface | null> {
   let providedWeb3: Web3;
-  if (config.nodeUrlEnvVar) {
-    const nodeUrl = process.env[config.nodeUrlEnvVar];
+  if (config.chainId && Number.isInteger(Number(config.chainId))) {
+    const nodeUrl = process.env[`NODE_URL_${config.chainId}`];
     if (!nodeUrl) throw Error(`Expected node url to be provided in env variable ${config.nodeUrlEnvVar}`);
```

Comment:
```suggestion
    if (!nodeUrl) throw Error(`Expected node url to be provided in env variable NODE_URL_${config.chainId}`);
```

Use `chainId` also in the error message